CHANGES.rst

```diff
@@ -1,8 +1,8 @@
 CHANGES
 =======
 
-2.3.9 (unreleased)
-------------------
+2.4 (unreleased)
```

Comment:
@thet this version bump is because of using ``get_portal``? Does this mean that p.a.users is only 5.1 compatible or CMFPlone 5.0 already has it?